frontend/src/libs/components/checkbox/checkbox.tsx

```diff
@@ -1,17 +1,37 @@
+import { useCallback, useState } from '#libs/hooks/hooks.js';
+
 import styles from './styles.module.scss';
 
 type Properties = {
   label: string;
+  groupName?: string;
   onChange: () => void;
+  type?: 'checkbox' | 'radio';
```

Comment:
checkbox and radio are 2 completely different elements, let's separate them

Reply:
done